packages/manager/apps/web/client/app/domain/webhosting/order/domain-webhosting-order.html

```diff
@@ -0,0 +1,22 @@
+<div data-ovh-alert="webhosting-order-alert"></div>
+
+<h2>
+    <span data-translate="domain_webhosting_order_title"></span>
```

Comment:
The `translate` directive could be applied on the parent `h2` element right?

Reply:
`h2` tag will be replace by a `oui-back-button`